flow_action_components/GetOpportunityPartners/force-app/main/default/classes/GetOpportunityPartnersAction.cls

```diff
@@ -0,0 +1,132 @@
+/**
+* @author       : Tamar Erlich
+* @date         : October 07, 2020
+* @description  : Invocable method that given opportunityId and accountId, returns a list of opportunity partners to display on opportunity plan record page
+* Note          : This class is called by the Flow.
+* *************************************************
+*       <runTest>GetOpportunityPartnersActionTest<runTest>
+* *************************************************
+* @History
+* -------
+* VERSION | AUTHOR                | DATE               | DESCRIPTION
+* 1.0     | Tamar Erlich          | October 07, 2020   | Created
+**/
+global with sharing class GetOpportunityPartnersAction {
+
+    // Attributes passed in from the Flow
+    global class Requests {
+
+        @InvocableVariable(label='OpportunityId' required=true)
```

Comment:
When writing labels we generally try to make use of spaces and emphasize readability, rather than just repeat the name of the underlying unique name.